### PR TITLE
(PDB-2006) Move the facts dual-db test to postgres

### PR DESCRIPTION
**Pull request by rbrw**

### CODE REVIEW

**Comment by rbrw** [comment]
This will be WIP until https://github.com/puppetlabs/puppetdb/pull/1663 makes it into master.

**Comment by ajroetker** [comment]
I think we should also move sync to use the second *sync* PuppetDB to use this configuration as well. That way they don't diverge.

File: test/puppetlabs/puppetdb/http/facts_test.clj [hunk 1]
```diff
@@ -581,73 +575,74 @@
...
+         (jdbc/with-transacted-connection write-db
+           (scf-store/add-certname! "foo1")
+           (scf-store/add-facts! {:certname "foo1"
+                                  :values facts1
+                                  :timestamp (now)
+                                  :environment "DEV"
+                                  :producer_timestamp (now)}))
+
+         (testing "queries only use the read database"
+           (let [request (get-request endpoint (json/parse-string nil))
```
Comment (wkalt) [inline test/puppetlabs/puppetdb/http/facts_test.clj]: prexisting, but I think these should all work fine with a single endpoint argument

**Comment by ajroetker** [comment]
I'm +1 here, but this is gonna break sync tests so I think we're gonna need to have a patch up there as well before this can get merged.

**Comment by kbarber** [comment]
@rbrw this might now need to be rebased against stable? Since master is now 4.0.0.

**Comment by rbrw** [comment]
@kbarber I think this should be up to date

**Pull request merged**